src/shidai/utils/joinerManager/tomlConfigurator.go

```diff
@@ -0,0 +1,41 @@
+package joinermanager
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	tomlEditor "shidai/utils/TomlEditor"
+	utilsTypes "shidai/utils/types"
+)
+
+func ApplyJoinerTomlSettings(sekaidHome string, tc *tomlEditor.TargetSeedKiraConfig, cfg *utilsTypes.ShidaiConfig) error {
+	ctx := context.Background()
+
+	client := &http.Client{}
+	info, err := tomlEditor.RetrieveNetworkInformation(ctx, client, tc)
```

Comment:
This is not a good idea (I understand that is my part of the code, but...) to leave logic of retrieving information about network in __TOML EDITOR__ package

Reply:
What about splitting them?